### PR TITLE
Fix ToC  discrepancies within Contributor Cheatsheets.

### PR DESCRIPTION
**Pull request by mrbobbytables**
Fixes several ToC issues across the contributor cheatsheet
/cc @kubernetes/sig-docs-fr-reviews 
/cc @kubernetes/sig-docs-hi-reviews 
/cc @kubernetes/sig-docs-id-reviews

### CODE REVIEW

**Comment by mrbobbytables** [comment]
Directly assigning 👍
This should mostly be a no-op. Just ensured titles/ToC were accurate, but want to be sure.
FR:
 /assign @remyleone 

HI:
/assign @Rajakavitha1 

ID:
/assign @irvifa

File: contributors/guide/contributor-cheatsheet/README-hi.md [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
+- [सहायक संसाधन](#सहायक-संसाधन)
+  - [आरंभ करना](#आरंभ-करना)
+  - [SIGS और अन्य समूह](#SIGS-और-अन्य-समूह)
+  - [समुदाय](#समुदाय)
+  - [महत्वपूर्ण ईमेल उपनाम](#महत्वपूर्ण-ईमेल-उपनाम])
+  - [वर्कफ़्लो](#वर्कफ़्लो)
+  - [परीक्षण](#परीक्षण)
+- [GitHub पर प्रभावी रूप से संवाद](#GitHub-पर-प्रभावी-रूप-से-संवाद)
+  - [अन्य उपयोगी लिंक्स](#अन्य-उपयोगी-लिंक्स)
+  - [एक-दूसरे से उत्कृष्ट कैसे बनें](#एक-दूसरे-से-उत्कृष्ट-कैसे-बनें)
```
Comment (vishakhanihore) [inline contributors/guide/contributor-cheatsheet/README-hi.md]: @mrbobbytables I think this should `एक-दूसरे के लिए उत्कृष्ट कैसे बनें`

File: contributors/guide/contributor-cheatsheet/README-hi.md [hunk 2]
```diff
@@ -1,35 +1,36 @@
...
+    - [अच्छे / बुरे संचार के उदाहरण](#अच्छे-/-बुरे-संचार-के-उदाहरण)
+- [कंट्रीब्यूशन सबमिट करना](#कंट्रीब्यूशन-सबमिट-करना)
+  - [CLA पर हस्ताक्षर करना](#CLA-पर-हस्ताक्षर-करना)
+  - [मुद्दों को खोलना और जवाब देना](#मुद्दों-को-खोलना-और-जवाब-देना)
+    - [एक मुद्दा बनाना](#एक-मुद्दा-बनाना)
+    - [एक मुद्दे पर प्रतिक्रिया](#एक-मुद्दे-पर-प्रतिक्रिया)
+  - [एक Pull Request खोलना](#एक-Pull-Request-खोलना)
+    - [एक Pull Request बनाना](#एक-Pull-Request-बनाना])
+    - [उदाहरण PR विवरण](#उदाहरण-PR-विवरण)
+    - [एक Pull Request समस्या निवारण](#एक-Pull-Request-समस्या-निवारण)
```
Comment (vishakhanihore) [inline contributors/guide/contributor-cheatsheet/README-hi.md]: `Pull Request की समस्या निवारण करना ` should be the title.

File: contributors/guide/contributor-cheatsheet/README-hi.md [hunk 3]
```diff
@@ -1,35 +1,36 @@
...
+  - [लेबल](#लेबल)
+- [स्थानीय स्तर पर काम करना](#स्थानीय-स्तर-पर-काम-करना)
+  - [शाखा रणनीति](#शाखा-रणनीति)
+    - [Upstream जोड़ना](#Upstream-जोड़ना)
+    - [अपने Fork को सिंक में रखना](#अपने-Fork-को-सिंक-में-रखना)
```
Comment (vishakhanihore) [inline contributors/guide/contributor-cheatsheet/README-hi.md]: `अपने Fork को समक्रमण में रखना `

File: contributors/guide/contributor-cheatsheet/README-hi.md (hunk 1 above)
Comment (mrbobbytables) [reply contributors/guide/contributor-cheatsheet/README-hi.md]: suggestions applied to all 3 👍

**Comment by irvifa** [comment]
Thanks Bob!
/lgtm

**Pull request merged**